.github/CODEOWNERS

```diff
@@ -22,11 +22,14 @@
 go.mod @grafana/backend-platform
 go.sum @grafana/backend-platform
 
-#Cloud Datasources backend code
+# Cloud Datasources backend code
 /pkg/tsdb/cloudwatch @grafana/cloud-datasources
 /pkg/tsdb/azuremonitor @grafana/cloud-datasources
 /pkg/tsdb/cloudmonitoring @grafana/cloud-datasources
 
+# Observability backend code
+/pkg/tsdb/influxdb @grafana/observability-squad
```

Comment:
Not sure if this should be exhaustive update or not, but if the former than any data source observability have in `Core datasources` can be also with it's backend. And Tempo which seems to be missing altogether.

Reply:
@aocenas Cool, updated